dev/TreeView/InteractionTests/TreeViewTests.cs

```diff
@@ -2688,6 +2688,61 @@ public void TreeViewRootNodeBindingTest()
             }
         }
 
+        [TestMethod]
```

Comment:
Looks like the InteractionTests.TreeViewTests.TreeViewSwappingNodesTest_ContentMode test regressed.

Reply:
> Looks like the InteractionTests.TreeViewTests.TreeViewSwappingNodesTest_ContentMode test regressed.

Just saw your reviews...
The test is setting ItemsSource to null and I was accessing itemsSource.Count without any check, which caused crash, should be fixed now.